src/photos.cs

```diff
@@ -1471,4 +1495,55 @@ interface PHLocalIdentifierMapping
 		[NullAllowed, Export ("error")]
 		NSError Error { get; }
 	}
+
+	[TV (16,0), Mac (13,0), iOS (16,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PHPersistentChange
+	{
+		[Export ("changeToken")]
+		PHPersistentChangeToken ChangeToken { get; }
+
+		[Export ("changeDetailsForObjectType:error:")]
+		[return: NullAllowed]
+		PHPersistentObjectChangeDetails ChangeDetails (PHObjectType objectType, [NullAllowed] out NSError error);
```

Comment:
English is fun... I don't think "Change" is a verb here.

```suggestion
		PHPersistentObjectChangeDetails GetChangeDetails (PHObjectType objectType, [NullAllowed] out NSError error);
```

Reply:
Fixed.